### ISSUE TITLE
Support pointer type in schema

### ISSUE THREAD

**Issue by efirs** [tigrisdata/tigris-client-go#288]
To support nullable types, the model:
```
type User struct {
  Address Address
  MailingAddress *Address
}
```
Should produce the schema:
```
{"Address": {"type": "object", ....}
{"MailingAddress": {"type": ["null", "object"], ...}
```

**Comment by efirs** [comment]
Currently both produce:
```
{"Address": {"type": "object", ....}
{"MailingAddress": {"type": "object", ...}
```

**Comment by efirs** [comment]
Fixed by #319
